command/recorderd/recorderd.conf.sample

```diff
@@ -35,7 +35,11 @@ M.chain = "local"
 
 -- scheulde time recorderd runs
 -- time is denoted by hour:minute, hour is denoted in 24-hour clock format
--- multiple time segment is separated by comma, e.g. "13:00-14:05, 15:20-16:34"
+-- hour ranges from 0 - 24, minute ranges from 0 - 60
```

Comment:
does it really handle times like 24:01  or 09:60? are these the same as 00:01 and 10:00?

Reply:
It handles 24:00 but 24:01 will be regarded as invalid number and discard this time slice.

The reason 24:00 exist is to specify continuous time rage, e.g.
Monday 7:00-24:00
Tuesday 0:00 - 7:00
will be translated to Monday 7:00 - Tuesday 7:00